archer init -- panic

Trying to run archer workflow again from scratch, and getting segfault:
```
~trivia-backend(master)$ archer init
Note: It's best to run this command in the root of your workspace.
Welcome the the ECS CLI! We're going to walk you through some questions to help you get set up
with a project on ECS. A project is a collection of containerized applications (or micro-services)
that operate together.

Looks like you have some projects already.
? Would you like to use one of your existing projects? Yes
Ok, here are your existing projects.
? Which one do you want to add a new application to? reinvent-argh
? Which type of infrastructure pattern best represents your application? Load Balanced Web App
? What do you want to call this Load Balanced Web App? trivia-backend
? Which Dockerfile would you like to use for trivia-backend app? Dockerfile
Ok great, we'll set up a Load Balanced Web App named trivia-backend in project reinvent-argh.
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x177afef]

goroutine 1 [running]:
github.com/aws/amazon-ecs-cli-v2/internal/pkg/cli.(*InitProjectOpts).Execute(0xc0000f43c0, 0xc000010018, 0x1976c6d)
	/Users/hsinghui/workspace/amazon-ecs-archer/internal/pkg/cli/project_init.go:107 +0x3f
github.com/aws/amazon-ecs-cli-v2/internal/pkg/cli.(*InitOpts).Run(0xc000142310, 0x1956472, 0x6)
	/Users/hsinghui/workspace/amazon-ecs-archer/internal/pkg/cli/init.go:118 +0x3fd
github.com/aws/amazon-ecs-cli-v2/internal/pkg/cli.BuildInitCmd.func2(0xc0000cbb80, 0x20cabc0, 0x0, 0x0, 0x0, 0x0)
	/Users/hsinghui/workspace/amazon-ecs-archer/internal/pkg/cli/init.go:181 +0x70
github.com/spf13/cobra.(*Command).execute(0xc0000cbb80, 0x20cabc0, 0x0, 0x0, 0xc0000cbb80, 0x20cabc0)
	/Users/hsinghui/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:826 +0x460
github.com/spf13/cobra.(*Command).ExecuteC(0xc00024c500, 0x0, 0x0, 0xc0000300b8)
	/Users/hsinghui/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:914 +0x2fb
github.com/spf13/cobra.(*Command).Execute(...)
	/Users/hsinghui/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:864
main.main()
	/Users/hsinghui/workspace/amazon-ecs-archer/cmd/archer/main.go:25 +0x27
```

Update: Nuked the project  parameter from systems manager and still getting the same error:
```
trivia-backend(master)$ archer init
Note: It's best to run this command in the root of your workspace.
Welcome the the ECS CLI! We're going to walk you through some questions to help you get set up
with a project on ECS. A project is a collection of containerized applications (or micro-services)
that operate together.

Looks like you don't have any existing projects. Let's create one!
? What would you like to call your project? reinvent-goose
? Which type of infrastructure pattern best represents your application? Load Balanced Web App
? What do you want to call this Load Balanced Web App? backend
? Which Dockerfile would you like to use for backend app? Dockerfile
Ok great, we'll set up a Load Balanced Web App named backend in project reinvent-goose.
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x177afef]

goroutine 1 [running]:
github.com/aws/amazon-ecs-cli-v2/internal/pkg/cli.(*InitProjectOpts).Execute(0xc0000d8370, 0xc000010018, 0x1976c6d)
	/Users/hsinghui/workspace/amazon-ecs-archer/internal/pkg/cli/project_init.go:107 +0x3f
github.com/aws/amazon-ecs-cli-v2/internal/pkg/cli.(*InitOpts).Run(0xc000146380, 0x1956472, 0x6)
	/Users/hsinghui/workspace/amazon-ecs-archer/internal/pkg/cli/init.go:118 +0x3fd
github.com/aws/amazon-ecs-cli-v2/internal/pkg/cli.BuildInitCmd.func2(0xc0002ae000, 0x20cabc0, 0x0, 0x0, 0x0, 0x0)
	/Users/hsinghui/workspace/amazon-ecs-archer/internal/pkg/cli/init.go:181 +0x70
github.com/spf13/cobra.(*Command).execute(0xc0002ae000, 0x20cabc0, 0x0, 0x0, 0xc0002ae000, 0x20cabc0)
	/Users/hsinghui/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:826 +0x460
github.com/spf13/cobra.(*Command).ExecuteC(0xc0000b7b80, 0x0, 0x0, 0xc000088058)
	/Users/hsinghui/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:914 +0x2fb
github.com/spf13/cobra.(*Command).Execute(...)
	/Users/hsinghui/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:864
main.main()
	/Users/hsinghui/workspace/amazon-ecs-archer/cmd/archer/main.go:25 +0x27

trivia-backend(master)$ 

```

--- a/internal/pkg/cli/init.go
+++ b/internal/pkg/cli/init.go
@@ -62,11 +62,13 @@ func NewInitOpts() (*InitOpts, error) {
 	}
 	prompt := prompt.New()
 	spin := spinner.New()
+	id := identity.New(sess)
 
 	initProject := &InitProjectOpts{
 		projectStore: ssm,
 		ws:           ws,
 		prompt:       prompt,
+		identity:     id,
 	}
 	initApp := &InitAppOpts{
 		fs:             &afero.Afero{Fs: afero.NewOsFs()},
@@ -83,7 +85,7 @@ func NewInitOpts() (*InitOpts, error) {
 		envDeployer:   cloudformation.New(sess),
 		prog:          spin,
 		prompt:        prompt,
-		identity:      identity.New(sess),
+		identity:      id,
 	}
 	return &InitOpts{
 		initProject: initProject,

